src/main/scala/util/DescribedSRAM.scala

```diff
@@ -46,7 +47,7 @@ object DescribedSRAM {
       depth = size,
       description = desc,
       write_mask_granularity = granWidth,
-      uid = DescribedSRAMIdAssigner.genId())
+      uid = id)
```

Comment:
This change seems worse than returning `id` from this function along with `mem`

Reply:
removed